packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -108,6 +109,25 @@ const sfdxMobilePreviewCommand = 'force:lightning:lwc:preview';
 const androidSuccessString = 'Launching... Opening Browser';
 
 export async function forceLightningLwcPreview(sourceUri: vscode.Uri) {
+  const preview = getPreview();
+  preview(sourceUri);
+}
+
+export function getPreview() {
+  if (isSFDXContainerMode()) {
+    return lwcPreviewContainerMode;
```

Comment:
@ryannmcq Whenever the command is run , do we always want to show the error message? No exceptions!

Reply:
@ryannmcq yes, that's what it sounds like from our product manager. Curious what possible exceptions you're envisioning though?

Reply:
Nothing else. Just wanted to confirm the same.